pysph/examples/rigid_body/rigid_block_in_tank.py

```diff
@@ -133,55 +140,60 @@ def create_particles(self):
             # exponent gamma
-            Group(equations=[
-                    TaitEOS(dest='fluid', sources=None, rho0=rho0, c0=c0, gamma=gamma),
-                    TaitEOSHGCorrection(dest='solid', sources=None, rho0=rho0, c0=c0, gamma=gamma),
-                    TaitEOSHGCorrection(dest='block', sources=None, rho0=rho0, c0=c0, gamma=gamma),
-                    ], ),
+            Group(
+                equations=[
+                    TaitEOS(dest='fluid', sources=None, rho0=rho0, c0=c0,
+                            gamma=gamma),
+                    TaitEOSHGCorrection(dest='solid', sources=None, rho0=rho0,
+                                        c0=c0, gamma=gamma),
+                    TaitEOSHGCorrection(dest='block', sources=None, rho0=rho0,
+                                        c0=c0, gamma=gamma),
+                ], ),
 
             # Main acceleration block
             Group(equations=[
 
-                    # Continuity equation with dissipative corrections for fluid on fluid
-                    ContinuityEquationDeltaSPH(dest='fluid', sources=['fluid'], c0=c0, delta=0.1),
-                    ContinuityEquation(dest='fluid', sources=['solid', 'block']),
-                    ContinuityEquation(dest='solid', sources=['fluid']),
-                    ContinuityEquation(dest='block', sources=['fluid']),
-
-                    # Momentum equation
-                    MomentumEquation(dest='fluid', sources=['fluid', 'solid', 'block'],
-                                     alpha=alpha, beta=beta, gy=-9.81, c0=c0,
-                                     tensile_correction=True),
-
-                    PressureRigidBody(dest='fluid', sources=['block', 'solid'], rho0=rho0),
-                    ViscosityRigidBody(dest='fluid', sources=['block', 'solid'], rho0=rho0, nu=nu),
-
-                    # Position step with XSPH
-                    XSPHCorrection(dest='fluid', sources=['fluid']),
-
-                    RigidBodyCollision(
-                        dest='block', sources=['solid'], k=1.0, d=2.0, eta=0.1, kt=0.1
-                    ),
-
-                    ]),
+                # Continuity equation with dissipative corrections for fluid on fluid
+                ContinuityEquationDeltaSPH(dest='fluid', sources=['fluid'],
+                                           c0=c0, delta=0.1),
+                ContinuityEquation(dest='fluid', sources=['solid', 'block']),
+                ContinuityEquation(dest='solid', sources=['fluid']),
+                ContinuityEquation(dest='block', sources=['fluid']),
+
+                # Momentum equation
+                MomentumEquation(dest='fluid',
+                                 sources=['fluid', 'solid', 'block'],
+                                 alpha=alpha, beta=beta, gy=-9.81, c0=c0,
+                                 tensile_correction=True),
+                PressureRigidBody(dest='fluid', sources=['block', 'solid'],
```

Comment:
Well if the example is not useful and produces bad results do we really need it?